### PR TITLE
Upgrade to 74637287fd52f5ccd8c97b816c7e8bb9b3bbb2ec, bump runtime ver…

### PR DESCRIPTION
**Pull request by Teeed**
…sion, remove deps to imbe_vocoder, bump version

### CODE REVIEW

**Comment by AsciiWolf** [comment]
Thanks! Do D-Star (AMBE) voice works without the imbe vocoder module?

File: io.github.nostar.DroidStar.metainfo.xml [hunk 1]
```diff
@@ -41,7 +41,7 @@
     <content_attribute id="social-chat">mild</content_attribute>
   </content_rating>
   <releases>
-    <release version="1.0.0-git" date="2023-03-19"/>
+    <release version="1.0.1-git" date="2023-03-19"/>
```
Comment (AsciiWolf) [inline io.github.nostar.DroidStar.metainfo.xml:44]: Please also bump the release date based on the commit used. Thanks!

**Comment by Teeed** [comment]
IMBE codec is now included in the project. For AMBE separate library is still needed (vocoder plugin).

**Comment by AsciiWolf** [comment]
Thanks! :-)

**Pull request merged**